app/about/ai.html

```diff
@@ -126,6 +127,21 @@ <h3 id="open-french-law-chatbot">Open French Law Chatbot</h3>
           <li>An AI-ready open French law dataset</li>
         </ul>
 
+        <h3 id="open-legal-ai-workbench">Open Legal AI Workbench (OLAW)</h3>
+        <p>
+          We want academics and nonprofits at the table in discovering the next generation of legal interfaces and helping to close the justice gap. It is not at all clear yet which legal AI tools and interfaces will work effectively for people with different levels of skill, what kind of guardrails they need, and what kind of matters they can help with.
+        </p>
+
+        <p>
+          That’s why we’re releasing OLAW, a common framework for scholarly researchers to build novel interfaces and run experiments. In technical terms, OLAW is a simple, well-documented, and extensible framework for legal AI researchers to build services using tool-based retrieval augmented generation.
```

Comment:
```suggestion
          That’s why we’re releasing OLAW, a common framework for scholarly researchers to build novel interfaces and run experiments. In technical terms, OLAW is a simple, well-documented, and extensible framework for legal AI researchers to build services using tool-based retrieval-augmented generation.
```
I see the phrase both with and without a hyphen, but I think it looks more correct with.